visualmetrics.py

```diff
@@ -141,9 +141,9 @@ def extract_frames(video, directory, full_resolution, viewport):
                    os.path.join(dir_escaped, 'img-%d.png')]
         logging.debug(' '.join(command))
         lines = []
-        proc = subprocess.Popen(command, stderr=subprocess.PIPE)
+        proc = subprocess.Popen(command, stderr=subprocess.PIPE, universal_newlines=True)
         while proc.poll() is None:
-            lines.extend(iter(proc.stderr.readline, ""))
+            lines.append(proc.stderr.readline())
```

Comment:
Can you revert this change as well? The existing code reads all of the pending lines in one pass.

Reply:
I encountered deadlock when program reach here

conda 4.10.1
python 3.8.8